change/@passageidentity-passage-node-6d33b410-664e-4e55-ae79-8f5199f35df5.json

```diff
@@ -0,0 +1,7 @@
+{
+    "type": "patch",
+    "comment": "SDK updates",
+    "packageName": "@passageidentity/passage-node",
+    "email": "bert.ramirez@agilebits.com",
+    "dependentChangeType": "patch"
+}
```

Comment:
@bertrmz I think you need to make the change as `none` because we want this to release with the changes already in main as a single minor change 2.3.0.

Reply:
My understanding of beachball is that it'll roll up all changes into a single bump.
So this could be `minor`, `patch`, or `none`, & the next minor change would still be the same.

I'll update it to `none`, but by semver this PR might even be a `minor`, because a new field is added to the App type.